priv/templates/extended_bin_windows_ps

```diff
@@ -305,7 +304,6 @@ function Uninstall() {
     }
     # Uninstall the Windows service
     & $erlsrv remove $service_name
-    & $epmd -kill
```

Comment:
Let me know if this makes sense to remove.

Reply:
I don't know why it was there in the first place. I guess it's to avoid leaving things behind after we're done with the execution, but that wouldn't be nice if we didn't start EPMD ourselves?

Reply:
@reganheath do you remember why this was added? I think this came with your big refactor of the windows scripts

Reply:
I wouldn't remove something unless it is known to cause an issue or to not be needed. We don't seem to know either of those at this time, right?

Reply:
@ferd I honestly don't recall why I added it.  I suspect you're correct and I was cleaning up under the (mistaken) assumption that it would be the only erlang process on the system.  

I think we could remove it, based on that.